civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,195 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom: 0;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0;
+    position: absolute;
+    top: 0;
+  }
+
+  .views-field-image-URL img {
+    border-radius: 50%;
+    height: 80px;
+    width: 80px;
+  }
+
+  .chr_panel__footer {
+    border-top: 0;
+    margin-top: 0;
+  }
+
+  .views-field-nothing {
+    .field-content {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      white-space: nowrap;
+      width: 100%;
+    }
+
+    .contact-id,
+    .gender,
+    .marital-status {
+      margin-right: 1rem;
+    }
+  }
+
+  .views-field-ni-ssn-70 .field-content {
+    width: 100%;
+  }
+
+  .nsi-exists.nsi-application-in-progress-1 {
```

Comment:
`nsi-exists` should follow BEM, see previous comment.

Reply:
What is this class `nsi-application-in-progress-1`?

Reply:
This Is to be able to theme different states for when NSI application is in progress or not. `1` is just a value appended to `nsi-application-in-progress`.